common/mariadb-galera/helm/.helmignore

```diff
@@ -0,0 +1,24 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+*.gotmpl
+custom/
+templates/secrets.json
```

Comment:
Ignores that are specific to your editor does not belong in the repo itself. Please configure `git config --global core.excludesFile` to set up your system-wide gitignores.

Reply:
That is a Helm ignore file. As far as i know Helm does not support a global/user ignore file and btw. we are using the same file also in [other charts](https://github.com/search?q=repo%3Asapcc%2Fhelm-charts%20helmignore&type=code).